npm/css-src/sass/overrides/_progress.scss

```diff
@@ -0,0 +1,32 @@
+.progress-display[value] {
```

Comment:
This file should probably be in the sass folder rather than the overrides folder. I think /overrides is for where we're overriding basic HTML defaults or the design system, whereas this is totally new code. 🙂